.travis.yml

```diff
@@ -1,17 +1,24 @@
 language: android
 jdk: oraclejdk8
 before_install:
-- nvm install 6
+- nvm install 8
 - node --version
 install:
-- cd examples && cd RNOneSignal && npm install
+- cd examples
+- cd RNOneSignal
+- npm install -g yarn
+- yarn
 android:
   components:
   - build-tools-23.0.1
```

Comment:
I think we can clean up this old `build-tools-23.0.1` version as well as `build-tools-27.0.3`.

Reply:
I don't believe, since I encountered errors while trying to do so.